awswrangler/typing.py

```diff
@@ -192,6 +193,13 @@ class TimestreamBatchLoadReportS3Configuration(TypedDict):
     """
 
 
+class ArrowDescryptionConfiguration(TypedDict):
+    """Configuration for decrypting encrypted columns."""
+
+    crypto_factory: pyarrow.parquet.encryption.CryptoFactory
+    kms_connection_config: pyarrow.parquet.encryption.KmsConnectionConfig
```

Comment:
We should document these two parameters here:
```python
    crypto_factory: pyarrow.parquet.encryption.CryptoFactory
    """Crypto factory for encrypting and decrypting columns."""
    kms_connection_config: pyarrow.parquet.encryption.KmsConnectionConfig
    """Configuration of the connection to the Key Management Service (KMS)."""
```

Reply:
Please clarify in the params docs that they cannot be used in distributed mode (i.e. when using Ray)